doc/source/data/creating-datasets.rst

```diff
@@ -304,6 +305,26 @@ are supported for each of these storage systems.
     :start-after: __read_parquet_az_begin__
     :end-before: __read_parquet_az_end__
 
+Reading from Local Storage
+==========================
+
+In Ray Datasets, users often read from remote storage systems as described above. In
+some use cases users may want to read from local storage. There are three ways to read
+from local filesystem:
+
+* Providing a local filesystem path: For example, in ``ray.data.read_csv("my_file.csv")``,
+  the given path will be resolved as a local filesystem path. Note: if the file exists
+  only on the local node and you run this read operation in distributed cluster, this
+  will fail as it cannot access the file from remote node.
```

Comment:
Nit: Use note directive.
```suggestion
  the given path will be resolved as a local filesystem path.
  .. note:: If the file exists only on the local node and you run this read
    operation in a distributed cluster, this will fail as it cannot access the
    file from a remote node.
```